tests/robot-cases/Group1-Nightly/Clair.robot

```diff
@@ -143,3 +143,56 @@ Test Case - Project Level Image Serverity Policy
     Set Vulnerabilty Serverity  0
     Cannot pull image  ${ip}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}  project${d}  ${image}  tag=${sha256}
     Close Browser
+
+Test Case - Verfiy System Level CVE Whitelist
+    [Tags]  run-once
+    Init Chrome Driver
+    ${d}=    Get Current Date    result_format=%m%S
+    ${image}=    Set Variable    postgres
+    ${sha256}=  Set Variable  68b49a280d2fbe9330c0031970ebb72015e1272dfa25f0ed7557514f9e5ad7b7
+    ${signin_user}=    Set Variable  user025
+    ${signin_pwd}=    Set Variable  Test1@34
+    Sign In Harbor    ${HARBOR_URL}    ${signin_user}    ${signin_pwd}
+    Create An New Project    project${d}
+    Push Image    ${ip}    ${signin_user}    ${signin_pwd}    project${d}    ${image}    sha256=${sha256}  tag=${sha256}
+    Pull Image    ${ip}    ${signin_user}    ${signin_pwd}    project${d}    ${image}    tag=${sha256}
+    Go Into Project  project${d}
+    Set Vulnerabilty Serverity  0
+    Cannot Pull image    ${ip}    ${signin_user}    ${signin_pwd}    project${d}    ${image}    tag=${sha256}
+    Go Into Project  project${d}
+    Go Into Repo  project${d}/${image}
+    Scan Repo  ${sha256}  Succeed
+    Logout Harbor
+    Sign In Harbor    ${HARBOR_URL}    ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}
+    Switch To Configure
+    Switch To Configuration System Setting
+    Create A New System CVE Whitelist    CVE-2019-12900\nCVE-2017-14062\nCVE-2019-11068\nCVE-2016-9427\nCVE-2018-6551\nCVE-2018-1000001\nCVE-2018-6485\nCVE-2019-9169\nCVE-2016-2779\nCVE-2019-13917\nCVE-2017-12424
+    Cannot Pull image    ${ip}    ${signin_user}    ${signin_pwd}    project${d}    ${image}    tag=${sha256}
+    Create A New System CVE Whitelist    CVE-2019-8457
+    Pull Image    ${ip}    ${signin_user}    ${signin_pwd}    project${d}    ${image}    tag=${sha256}
+    Close Browser
+
+Test Case - Verfiy Project Level CVE Whitelist
+    [Tags]  run-once
+    Init Chrome Driver
+    ${d}=    Get Current Date    result_format=%m%S
+    ${image}=    Set Variable    redis
+    ${sha256}=  Set Variable  9755880356c4ced4ff7745bafe620f0b63dd17747caedba72504ef7bac882089
+    ${signin_user}=    Set Variable  user025
+    ${signin_pwd}=    Set Variable  Test1@34
+    Sign In Harbor    ${HARBOR_URL}    ${signin_user}    ${signin_pwd}
+    Create An New Project    project${d}
+    Push Image    ${ip}    ${signin_user}    ${signin_pwd}    project${d}    ${image}    sha256=${sha256}  tag=${sha256}
+    Pull Image    ${ip}    ${signin_user}    ${signin_pwd}    project${d}    ${image}    tag=${sha256}
```

Comment:
why pull it after pushing?

Reply:
Just to make sure image can be pulled before any configurations.